Client/Application/LeanplumIntegration.swift

```diff
@@ -10,6 +10,26 @@ import Leanplum
 private let LeanplumEnvironmentKey = "LeanplumEnvironment"
 private let LeanplumAppIdKey = "LeanplumAppId"
 private let LeanplumKeyKey = "LeanplumKey"
+private let applicationDidRequestUserNotificationPermissionPrefKey = "applicationDidRequestUserNotificationPermissionPrefKey"
+
+// FxA Custom Leanplum message template for A/B testing
+// push notifications.
+let LPMT_FXA_PRE_PUSH = "FxA Prepush v1"
+let LPMT_ARG_ACCEPT_ACTION = "Accept action"
+let LPMT_ARG_CANCEL_ACTION = "Cancel action"
+let LPMT_ARG_TITLE_TEXT = "Title.Text"
+let LPMT_ARG_TITLE_COLOR = "Title.Color"
+let LPMT_ARG_MESSAGE_TEXT = "Message.Text"
+let LPMT_ARG_MESSAGE_COLOR = "Message.Color"
+let LPMT_ARG_ACCEPT_BUTTON_TEXT = "Accept button.Text"
+let LPMT_ARG_CANCEL_BUTTON_TEXT = "Cancel button.Text"
+let LPMT_ARG_CANCEL_BUTTON_TEXT_COLOR = "Cancel button.Text color"
+
+// These defaults are overridden though Leanplum webUI
+let LPMT_DEFAULT_ASKT_TO_ASK_TITLE = "Firefox Sync Requires Push"
+let LPMT_DEFAULT_ASK_TO_ASK_MESSAGE = "Firefox will stay in sync faster with Push Notifications enabled."
+let LPMT_DEFAULT_OK_BUTTON_TEXT = "Enable Push"
+let LPMT_DEFAULT_LATER_BUTTON_TEXT = "Don't Enable"
```

Comment:
Can we use camel casing for these, as we do with the rest of our `NSLocalizableStrings`? Shouldn't these be `NSLocalizableString`s here too?